ecl/hql/hqllex.l

```diff
@@ -1044,6 +1044,8 @@ __OS__              {
                             return lookupIdentifierToken(returnToken, lexer, lookup, activeState, CUR_TOKEN_TEXT);
 #ifdef _WIN32
                         returnToken.setExpr(createConstant("windows"));
+#elif defined(__APPLE__)
+                        returnToken.setExpr(createConstant("linux-macos"));
```

Comment:
@richardkchapman is this a sensible value?

Reply:
Not really. MacOS is not linux or linux-based, though it shares some similarity. I don't see how returning this value helps any with back-compatibility - existing code is still going to have to change - so we may as well get it right.